samples/filterSnippets.js

```diff
@@ -0,0 +1,351 @@
+/**
```

Comment:
I know this is annoying, but could I trouble you to rebase and copy one of the existing license headers?  We made all of them consistent :)